gddo-server/main.go

```diff
@@ -898,7 +900,17 @@ func main() {
 
 	cacheBusters.Handler = mux
 
-	if err := http.ListenAndServe(*httpAddr, hostMux{{"api.", apiMux}, {"", mux}}); err != nil {
+	var allMux http.Handler = hostMux{{"api.", apiMux}, {"", mux}}
```

Comment:
The mux name is already taken by the `{"", mux}` subdomain.

I could rename that to webMux or something, but that was a slightly larger change. Totally up for it, though.